src/main/java/org/folio/config/ApplicationConfig.java

```diff
@@ -105,6 +109,36 @@
 @ComponentScan({ "org.folio" })
 public class ApplicationConfig {
 
+  private static final Logger LOGGER = LogManager.getLogger();
+
+  @Value("${KAFKA_HOST:kafka}")
```

Comment:
nitpick: I think this should default to localhost.

Reply:
the default values used for the container in vagrant

Reply:
sounds good.